packages/contentful/migrations/externalAuthors/20221005103626-create-external-authors.js

```diff
@@ -0,0 +1,89 @@
+module.exports.description = 'Create content model for External Authors';
```

Comment:
as specified in the ticket - at this stage we are only supposed to create a POC entity. I think what you did was convert all the existing content types into migrations but you shouldn't have. They were just there as a POC too and should be removed. 

Why don't you just keep one migration checked in for, lets say, news (which we know we are going to create next)?